From cffec7f607d550b1cf51a75315104559732ca343 Mon Sep 17 00:00:00 2001
From: Paul Harrison <pharrison@mythtv.org>
Date: Sat, 15 Mar 2014 16:31:51 +0000
Subject: [PATCH 1/5] MusicMetadata: copy m_actualFilename in operator=

This fixes the regression that was the root cause of #12088 and also a similar
problem in the import screen when trying to play the tracks there as well.

Refs #12088.
---
 mythtv/libs/libmythmetadata/musicmetadata.cpp |    1 +
 1 file changed, 1 insertion(+)

diff --git a/mythtv/libs/libmythmetadata/musicmetadata.cpp b/mythtv/libs/libmythmetadata/musicmetadata.cpp
index f4ebd1c..5a883c1 100644
--- a/mythtv/libs/libmythmetadata/musicmetadata.cpp
+++ b/mythtv/libs/libmythmetadata/musicmetadata.cpp
@@ -117,6 +117,7 @@ MusicMetadata& MusicMetadata::operator=(const MusicMetadata &rhs)
     m_compilation = rhs.m_compilation;
     m_id = rhs.m_id;
     m_filename = rhs.m_filename;
+    m_actualFilename = rhs.m_actualFilename;
     m_hostname = rhs.m_hostname;
     m_directoryid = rhs.m_directoryid;
     m_artistid = rhs.m_artistid;
-- 
1.7.10.2

